quasarnp/utils.py

```diff
@@ -112,7 +114,7 @@ def process_preds(preds, lines, lines_bal):
         c_line_bal[il] = preds[il + nlines][:, :13].max(axis=1)
         z_line_bal[il] = (i_to_wave((j + offset) * len(wave) / nboxes) / l) - 1
 
-    return c_line, z_line, zbest, c_line_bal, z_line_bal
+    return c_line, z_line, zbest, cbest, c_line_bal, z_line_bal
```

Comment:
@sbailey That is the confidence value for the line copied into zbest (the confidence of the highest confidence line found, zbest is the redshift of that line). I pulled it out of c_line explicitly so I was positive I was pulling the right one. I'll add a doc string to load_desi_coadd(), but, since I wrote it, no one has used it yet but me, thankfully.

Reply:
The only reason the return values looked like this was because I did not write `process_preds`, I actually ported it straight from QuasarNet and then refactored it to add some comments and make it more readable. So the return string is identical to how it appeared there. The only place I know that calls it is the README example code, which would need to be updated. Generally in my code I have been using `c_line, z_line, zbest, *_ = process_preds(*args)` when analyzing, so most of _my_ code would not break with this api change.

The only reason I would be apprehensive is because adding `cbest` would break interoperability with original QuasarNet code. Up to this point it is very easy to replace `from quasarnet.utils import process_preds` with `from quasarnp.utils import process_preds` without changing any other code and having it "just work". I do not know how much of a major concern maintaining this backwards/forward compat is. I will let @dkirkby and @sbailey decide if this is important enough to keep or not. 

If we elect to try and maintain the previous API, I think `cbest` can be computed easily from `c_line`. Personally though I have no strong feelings on keeping nor changing it.

Reply:
@dylanagreen that's a good point about maintaining API compatibility with existing QuasarNet, at least for now.

@eleanorlyke am I correct that `cbest` is completely derivable from the `c_line` array that is already returned?  i.e. although it is *convenient* to have it pre-calculated in the return values, it isn't *required* in order to access that information, correct?

Reply:
@sbailey Yes, you are correct. There were issues with low confidence values being pulled from c_lines, so I used this to get the one I was sure it was using out of c_lines. But turned out to be an indenting issue further upstream. I'll remove this change and revert the doc string to the original.